db/index.js

```diff
@@ -6,17 +6,16 @@ const {
   databaseDomain,
   databasePort,
   databaseName,
+  databaseURL,
   databaseUsername,
   databasePassword
 } = require('../config')
 const debug = require('debug')(`${appName}:db`) // DEBUG=your_app_name:db
 const chalk = require('chalk')
 const Sequelize = require('sequelize')
 const name = (databaseName || appName) + (isTesting ? '_test' : '')
-const url =
-  `postgres://${databaseUsername}:${databasePassword}@${databaseDomain}:${databasePort}/${databaseName}` ||
-  `postgres://@localhost:5432/${name}`
 
+const url = databaseURL || 'postgres://localhost:5432/mobvoice'
```

Comment:
My local configs are not being picked up with this change. I think you need to define databaseURL in the `config/index.js` file because it is currently always defaulting to the hardcoded value here.

Reply:
Fixed in commit 8edf803